integr8ly/manifests
# bump integreatly-operator to 1.5.4

## Verification
- Delete other integreatly operatorsources
- Create operatorsource pointing at `philbrookes`
- Install integreatly-operator in a namespace
- Create the required s3 secrets
- create a valid installation CR
- See the installation succeed and using the new status format.
